### DISCUSSION TITLE
Is there a way to bind the down/up arrow keys?

### DISCUSSION THREAD

**Discussion by malkomalko** [johnlindquist/kit#1547] [Q&A]
I find myself wanting to hit ctrl+j/k or ctrl+n/p instead of down/up arrows so I don't have to reach far away as scriptkit mostly navigates through menus. I haven't found a way to do this.

**Comment by johnlindquist** [comment]
Hmm, it's easy to add. Just curious how it would conflict with the "cmd/ctrl+k" for the actions menu. I guess we could do alt/option+j/k?

I usually recommend just binding caplock+j/k globally on your system so you can navigate anywhere
